tools/ci_build/build.py

```diff
@@ -1513,6 +1533,17 @@ def main():
     if args.skip_tests:
         args.test = False
 
+    if (args.include_ops_by_model and len(args.include_ops_by_model) > 0) or\
+       (args.include_ops_by_file and len(args.include_ops_by_file) > 0):
+
+        include_ops_by_model = args.include_ops_by_model if args.include_ops_by_model else ''
+        include_ops_by_file = args.include_ops_by_file if args.include_ops_by_file else ''
+
+        for ep_path in paths_to_execution_provider(args):
+            rewrite_provider(include_ops_by_model, include_ops_by_file, ep_path)
+
+        args.test = False #disable tests since we don't know which ops are enabled
+
```

Comment:
You'll need to also adjust cmake files to not attempt to build the operator unit tests.

Reply:
Isn't this sufficient for the purpose - in my case the test stuffs are not touched at all.